### DISCUSSION TITLE
Can the ConfirModal box cancel icon？

### DISCUSSION THREAD

**Discussion by Huoyubai114** [lowdefy/lowdefy#1150] [Questions and Help] [answered]
Hello, I'd like to ask if the ConfirModal box can cancel icon？Thank you very much!

id: block_id
type: ConfirmModal
properties:
  icon: AccountBookOutlined
  mask: false
  cancelButton:
    icon: AccountBookOutlined

**Comment by Gervwyk** [comment, marked as answer]
Hi @Huoyubai114 
I can confirm this is a bug. Will open an issue and fix it in the next patch release.
Thanks for bringing this to our attention.
